CHANGELOG.rst

```diff
@@ -49,6 +49,10 @@ Added
 - Add a `bot_challenge` intent and a `utter_iamabot` action to all example projects and the rasa init bot.
 - Allow sending attachments when using the socketio channel
 - ``rasa data validate`` will fail with a non-zero exit code if validation fails
+- The tracker is now available within the interpreter's ``parse`` method, giving the ability to create interpreter classes that
```

Comment:
Can be removed as it is duplicated. I guess, it is a leftover from a merge.

Reply:
Do I need to move this into the `1.4.0` section?

Reply:
No, it is already part of section [1.3.0] - 2019-09-05. The entry is just duplicated, so need to be removed here.